src/pages/monetization/partner-program-api/index.mdx

```diff
@@ -1590,6 +1591,53 @@ curl --request POST \
 
 </Section>
 
+<Section>
+
+<Text>
+
+## Get Summary
+
+`GET /v2/partners/affiliates/summary` - get a number of trials, paid and expired affiliate licenses
```

Comment:
I am wondering if this part is clear...🤔  If we mean that someone gets a number of both - trials and paid and explored affiliate licenses, I would change it to `get a number of trials and paid and expired affiliate licenses`. If someone can get trials or licenses, then I would change to `get a number of trials or paid and expired affiliate licenses`. WDYT?

Reply:
This endpoint returns trial affiliate licenses, paid affiliate licenses and expired affiliate licenses. Maybe I've lost the meaning in the translation :) What would be the correct description?

Reply:
so maybe, in that case, let's do it this way - `get a number of get a number of trial affiliate licenses, paid affiliate licenses, and expired affiliate licenses`
I think that this way it's the most clear :)